### PR TITLE
Geo distance and geo bounding box filter

### PR DESCRIPTION
**Pull request by impy88**
Thanks for the lib, It helps me so much!

please review this two filters I've added, I'll be happy if you find them useful or let me know if I missed something.

Cheers

### CODE REVIEW

File: src/filters/index.js [hunk 1]
```diff
@@ -26,6 +28,8 @@ export default {
   exists: existsFilter,
   exist: existsFilter,
   fuzzy: fuzzyFilter,
+  geo_bounding_box: geoBoundingBoxFilter,
+  geo_distance: geoDistanceFilter,
```
Comment (danpaz) [inline src/filters/index.js]: Could also add `geo-distance` and `geoDistance` as other matching strings for this filter. The goal is to make sure whatever the user guessed is likely to return the filter that they expected.

Comment (danpaz) [inline src/filters/index.js]: Same goes for `geo_bounding_box`: `geo-bounding-box` and `geoBoundingBox` should also work.

**Comment by danpaz** [comment]
@impy88 LGTM. Glad to hear you're finding the project useful - care to share what you're using it for? Always curious to hear how its solving other people's problems 😄 

Just a few small comments then I think this is good to go!

**Comment by impy88** [comment]
@danpaz Thanks, I've updated the PR.

Your lib is a quite useful when you use Couchbase + ElasticSearch stack (for IoT purposes), Couchbase's queries are a bit silly especially with fuzzy and geo queries. To get data from ES I don't want to write a huge JSON object (especially if needed chain-able solution ) or I don't need a monster like an ODM library. Here must be simple and fast solution like yours.

**Comment by danpaz** [comment]
Thanks @impy88! 

This PR also has me thinking that the barrier to using new query clauses that haven't been created yet in bodybuilder may be too steep. I wonder if it makes sense to provide an alternate api for providing your own query clauses in the case bodybuilder doesn't have a builder for it yet, something like `rawFilter` and `rawQuery`.

**Pull request merged**